<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>enterface</class>
 <widget class="QWidget" name="enterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>658</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="but_back">
     <property name="text">
      <string>上一步</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="but_next">
     <property name="text">
      <string>下一步</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QToolButton" name="but_data">
       <property name="text">
        <string>个人信息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="but_face">
       <property name="text">
        <string>录入人脸</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="but_verify">
       <property name="text">
        <string>验证结果</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="but_finish">
       <property name="text">
        <string>完成录入</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>学院：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cbB_college">
           <item>
            <property name="text">
             <string>电子与信息工程学院</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>大数据与软件工程学院</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>年级：</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="cbB_grade">
           <item>
            <property name="text">
             <string>2019</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2020</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2021</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2022</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2023</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2024</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2025</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>班级：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="le_class"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>学号：</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="le_ID"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>姓名：</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="le_name"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="1">
        <widget class="QLabel" name="lb_camera">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(205, 240, 255);</string>
         </property>
         <property name="text">
          <string>打开摄像头</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="lb_photo1">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(253, 255, 208);</string>
             </property>
             <property name="text">
              <string>照片1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lb_photo2">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(225, 227, 255);</string>
             </property>
             <property name="text">
              <string>照片2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lb_photo3">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 207, 208);</string>
             </property>
             <property name="text">
              <string>照片3</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="but_delet1">
             <property name="text">
              <string>删除</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="but_delet2">
             <property name="text">
              <string>删除</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="but_delet3">
             <property name="text">
              <string>删除</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="but_camera">
           <property name="text">
            <string>open camera</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="but_panorama">
           <property name="text">
            <string>全景模式</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="but_facecut">
           <property name="text">
            <string>人脸模式</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bur_screenshot">
           <property name="text">
            <string>拍照</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="but_save">
           <property name="text">
            <string>保存</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="but_cancel">
           <property name="text">
            <string>取消保存</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="lb_facedata">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(188, 255, 235);</string>
         </property>
         <property name="text">
          <string>人脸库里的人脸</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="QLabel" name="lb_camera2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 249, 210);</string>
         </property>
         <property name="text">
          <string>摄像头</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lb_print">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 206, 206);</string>
         </property>
         <property name="text">
          <string>打印相关信息</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_4">
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 181, 181);</string>
         </property>
         <property name="text">
          <string>完成添加人脸信息</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="but_nextpeople">
         <property name="text">
          <string>添加下一个</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="but_end">
         <property name="text">
          <string>结束添加</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
